data/json/items/gun/44.json

```diff
@@ -1,5 +1,34 @@
 [
   {
+    "id": "cane_rifle",
+    "type": "GUN",
+    "name": "walking cane",
+    "description": "Gun in a cane.  A custom-built .44 Magnum rifle carefully integrated into a walking cane.  It's also reinforced for hand-to-hand combat.  Never underestimate a handicapped person.",
+    "weight": 950,
+    "volume": 4,
+    "price": 400000,
+    "bashing": 15,
+    "to_hit": 2,
+    "material": [ "steel", "aluminum", "wood" ],
+    "symbol": "/",
+    "color": "dark_gray",
+    "skill": "rifle",
+    "ammo": "44",
+    "ranged_damage": 11,
```

Comment:
My bad, I thought you were talking about manipulating the "barrel length"
Stat to get higher damage.
The bonus derived from base barrel length should definitely have a cap, and
it's just a guideline, a gun with a compromised design like a gun cane
isn't going to be able to take full advantage of it due to issues with
rigidity and tolerances.